src/dom/DomEvent.js

```diff
@@ -23,7 +23,7 @@ import {getScale} from './DomUtil';
 // Adds a set of type/listener pairs, e.g. `{click: onClick, mousemove: onMouseMove}`
 export function on(obj, types, fn, context) {
 
-	if (typeof types === 'object') {
+	if (types && typeof types === 'object') {

```

Comment:
Should we add a check `if(!types)` because now it will go into `Util.splitWords(types);` and throws an error (if types == `undefined`)

Reply:
> and throws an error (if types == `undefined`)

Because that is exactly what is desired - to catch user bug.